lib/src/workmanager.dart

```diff
@@ -65,16 +65,13 @@ class Workmanager {
   static const String iOSBackgroundTask = "iOSPerformFetch";
   static bool _isInDebugMode = false;
 
-  static const MethodChannel _backgroundChannel = const MethodChannel(
-      "be.tramckrijte.workmanager/background_channel_work_manager");
-  static const MethodChannel _foregroundChannel = const MethodChannel(
-      "be.tramckrijte.workmanager/foreground_channel_work_manager");
+  static const MethodChannel _backgroundChannel = const MethodChannel("be.tramckrijte.workmanager/background_channel_work_manager");
```

Comment:
Is dit geformatted met dartfmt?

Reply:
It was indeed configured incorrectly at first. I reformatted with default Dart FMT settings